promptsource/templates/common_gen/templates.yaml

```diff
@@ -27,11 +43,64 @@ templates:
     metadata: !TemplateMetadata
       _do_eval: false
       _do_train: true
-      choices_in_prompt: null
-      metrics: []
-      original_task: false
+      choices_in_prompt: false
+      metrics:
+      - BLEU
+      - ROUGE
+      original_task: true
     name: choice in concept centric sentence generation
     reference: ''
+  b2033df4-7922-45b8-a113-e7784135cea9: !Template
+    answer_choices: null
+    answer_choices_key: null
+    id: b2033df4-7922-45b8-a113-e7784135cea9
+    jinja: "{% set seq = [ \n'From the concepts mentioned below, generate a sentence:',\
+      \ \n'Convert the concepts to a sentence:', \n'Given the list of concepts, write\
+      \ a sentence:'\n] %} \n{{ seq | random }}\n{{ concepts | join(\", \") }}\n|||\n\
+      {{target}}"
+    metadata: !TemplateMetadata
+      choices_in_prompt: false
+      metrics:
+      - BLEU
+      - ROUGE
+      original_task: true
+    name: random task template prompt
+    reference: With a randomly chosen task template, generate a sentence from the
+      list of concepts
+  b7012213-04c4-424d-85fb-39d63d8a0ca2: !Template
+    answer_choices: null
+    answer_choices_key: null
+    id: b7012213-04c4-424d-85fb-39d63d8a0ca2
+    jinja: 'What is the topic of the sentence: {{target}}
+
+      |||
+
+      {{ concepts | random }}'
```

Comment:
Should be choice, but why not just list them all? I don't like randomness in answers.